tools/ci_build/github/azure-pipelines/android-x86_64-crosscompile-ci-pipeline.yml

```diff
@@ -76,7 +76,9 @@ jobs:
   pool:
     vmImage: 'ubuntu-latest'
   dependsOn: Android_CI
-  condition: and(succeeded(), in(variables['Build.Reason'], 'IndividualCI', 'BatchedCI'))
+  # disable update dashboard of Android Code coverage until Azure CI machine cannot access database issue is resolved
```

Comment:
Can you please add a task to the dashboard to undo this so we don't forget (set it to In Progress so it shows up as being active).